content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst

```diff
@@ -0,0 +1,180 @@
+on the order itself. Alternatively, a check can be processed from the check's page.
+
+From the check's page
+---------------------
+
+To process a *Measure* quality check from the check's page, begin by navigating to
+:menuselection:`Quality --> Quality Control --> Quality Checks`, and select a quality check. Follow
+the :guilabel:`Instructions` for how to take the measurement.
+
+After taking the measurement, record the value in the :guilabel:`Measure` field on the quality check
+form.
+
+On an order
+-----------
+
+To process a *Measure* quality check on an order, select a manufacturing order or inventory order
+(receipt, delivery, return, etc.), for which a check is required. Manufacturing orders can be
+selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
+
+On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
+at the top of the page. Click the button to open the :guilabel:`Quality Check` pop-up window, which
+shows all of the quality checks required for that order.
+
+To process a *Measure* quality check, measure the product as instructed, then enter the value in the
+:guilabel:`Measure` field on the pop-up window. Finally, click :guilabel:`Validate` to register the
+recorded value.
+
+.. image:: measure_check/measure-check-pop-up.png
+   :align: center
+   :alt: A Measure quality check pop-up window on a manufacturing or inventory order.
+
+If the value entered is within the range specified in the :guilabel:`Tolerance` section of the
+|QCP|, the quality check passes and the pop-up window closes. The rest of the manufacturing or
+inventory order can then be processed as usual.
+
+However, if the value entered is outside of the specified range, a new pop-up window appears, titled
+:guilabel:`Quality Check Failed`. The body of the pop-up shows a warning message that states, "You
+measured # units and it should be between # units and # units.", as well as the instructions entered
+in the :guilabel:`Message If Failure` tab of the |QCP|. At the bottom of the pop-up, two buttons
+appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
+
+.. image:: measure_check/measure-check-failed.png
+   :align: center
+   :alt: The "Quality Check Failed" pop-up window.
+
+If the measurement is incorrect and should be changed, select :guilabel:`Correct Measure`. Doing so
+re-opens the :guilabel:`Quality Check` pop-up window. Enter the corrected measurement in the
+:guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check.
+
+If the measurement is correct, click :guilabel:`Confirm Measure` instead, and the quality check
+fails. Follow any instructions that were listed on the :guilabel:`Quality Check Failed` pop-up
+window.
```

Comment:
"If the measurement is correct, ... and the quality check fails" is a little confusing to me because the wording, "measurement is correct" implies the check should pass. Instead, what do you think of the wording:
"If the measurement was entered correctly,
if the measurement was not entered correctly,..." ?